src/main/java/homework_3/pyramidprinter/Main.java

```diff
@@ -0,0 +1,9 @@
+package homework_3.pyramidprinter;
```

Comment:
I thought these apps are in homework_2 package, and only UmmitableClass is in homework_3

Reply:
Fix that